app/services/form_validator.rb

```diff
@@ -60,6 +61,20 @@ def validate_country(form_element, el_name, errors)
     end
   end
 
+  def validate_postal(form_element, el_name, errors)
+    zip = @params[el_name]
+    country = @params.fetch(:country, :US)
+    if country.respond_to? :to_sym
+      country = country.to_sym
+    else
+      # country is likely to be nil, so set it to our default
```

Comment:
ahh gotcha. I suppose it could be an empty string too maybe, which would cast into the awkward `:""`. maybe do a blank check here instead, like 
`country = (country.blank? ? :US : country.to_sym)`